src/handlers/user_commands.ts

```diff
@@ -11,7 +11,10 @@ export const authHandler = (skipAskConnect=false) => {
 	userState.setWaitingForLogin();
 	vscode.env.openExternal(vscode.Uri.parse(authUrl));
 };
-
+export const requestDemoUrl = () => {
+	const authUrl = generateRequestDemoUrl();
+	vscode.env.openExternal(vscode.Uri.parse(authUrl));
```

Comment:
@ZahidSaeed01 After opening the URL, I think we should show a plugin-notification that user needs to Submit the form on Webapp. Thoughts?

Reply:
<img width="517" alt="Screenshot 2024-08-13 at 5 12 03 PM" src="https://github.com/user-attachments/assets/02e22038-c1bc-4f70-a66c-c230f3660d14">

Reply:
I requested to show the notification after user opens the URL (Clicks "Request a demo")
Notification should not contain any buttons I think. 
We can show a message, something like  "Please submit the Sign-Up Request Form on Webapp. We will notify you once your request is approved."